diff --git a/node_modules/@auth0/nextjs-auth0/dist/auth0-session/client/edge-client.js b/node_modules/@auth0/nextjs-auth0/dist/auth0-session/client/edge-client.js
index ce24e33..5cc4366 100644
--- a/node_modules/@auth0/nextjs-auth0/dist/auth0-session/client/edge-client.js
+++ b/node_modules/@auth0/nextjs-auth0/dist/auth0-session/client/edge-client.js
@@ -2,8 +2,8 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.clientGetter = exports.EdgeClient = void 0;
 const tslib_1 = require("tslib");
-const oauth = tslib_1.__importStar(require("oauth4webapi"));
-const jose = tslib_1.__importStar(require("jose"));
+const oauth = tslib_1.__importStar(import("oauth4webapi"));
+const jose = tslib_1.__importStar(import("jose"));
 const abstract_client_1 = require("./abstract-client");
 const errors_1 = require("../utils/errors");
 const errors_2 = require("../../utils/errors");
